crates/nargo/tests/test_data/main_return/Verifier.toml

```diff
@@ -1 +1 @@
-return = "8"
+return = "0x0000000000000000000000000000000000000000000000000000000000000008"
```

Comment:
Looks like running the tests locally now mangles the Verifier files

Reply:
we should remove verifier.toml from source control, they are generated during prove.